### PR TITLE
Refactored Travis caching

### PR DESCRIPTION
**Pull request by pamil**
- Changed custom cache directory from `$HOME/.mink-log-cache` to `$HOME/.mink-debug-cache`
- This path is available as environmental variable `$CACHE_DIR`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,15 @@
 language: php
 
+env:
+  global:
+    - CACHE_DIR=$HOME/.mink-debug-cache
```
Comment (michalmarcinkowski) [inline .travis.yml]: `CACHE_DIR` is very general, maybe something more specific `MINK_DEBUG_CACHE_DIR`?

Comment (pamil) [inline .travis.yml]: It is supposed to be general, what do you think about `$TRAVIS_CACHE_DIR`?

Comment (pjedrzejewski) [inline .travis.yml]: :+1: But then, maybe we should rename the default folder, right? On the other hand, we don't want to conflict with travis cache some day.

Comment (pamil) [inline .travis.yml]: I don't understand what you mean by the default folder. `$PROJECT_CACHE_DIR`?

Comment (pjedrzejewski) [inline .travis.yml]: No, if we call it `TRAVIS_CACHE_DIR`, then its default name should not be `.mink-debug-cache`, agree? :)

Comment (pamil) [inline .travis.yml]: Okay, let's stick to `MINK_DEBUG_CACHE_DIR`, it's quite long, but `SYLIUS_CACHE_DIR` makes sense in Sylius.

Comment (pjedrzejewski) [inline .travis.yml]: :+1:

Comment (michalmarcinkowski) [inline .travis.yml]: It's quite long, but there is a small possibility someone uses this name somewhere else.

**Comment by pjedrzejewski** [comment]
:+1:

**Pull request merged**